src/org/sipdroid/sipua/PublishAgent.java

```diff
@@ -109,16 +94,22 @@ public void publish(BasicStatus status, int expireTime, String note) {
 				e.printStackTrace();
 			}
 			String entity = "sip:" + user_profile.from_url;
-			String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
-							"<presence xmlns=\"urn:ietf:params:xml:ns:pidf\"" +
-							" entity=\"" + entity + "\">" +
-							"<tuple id=\"" + tupleId + "\">" +
-							"<status>" +
-							"<basic>" + status.pidf() + "</basic>" +
-							"<note>" + note + "</note>" +
-							"</status>" +
-							"</tuple>" +
-							"</presence>";
+			String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>" +
+					"<presence xmlns=\"urn:ietf:params:xml:ns:pidf\"\n" +

```

Comment:
you've added ```\n``` here.  SIP actually uses ```\r\n```, I'm not sure if that applies to the PIDF though, I think it is only for the headers.  Please remove ```\n``` on this line and the next line and put spaces instead